File: android/app/src/main/kotlin/com/example/hotel_reception/procesor/dto/update/RoomBusinessType.kt
package com.example.hotel_reception.procesor.dto.update

data class RoomBusinessType(val daysOccupation: Int, val type: RoomBusinessTypeEnum)

enum class RoomBusinessTypeEnum {
    occupiedByCustomer,
    dirty,
    cleaning,
    repairing,
    empty,
}

File: ios/Runner/Processor/dto/update/RoomBusinessType.swift
//
//  RoomBusinessType.swift
//  Runner
//
//  Created by Serhii Rakovskyi on 25.08.2022.
//

import Foundation

struct RoomBusinessType: Codable {
    let daysOccupation: Int
    let type: String
}
enum  RoomBusinessTypeEnum  : Codable {
    case occupiedByCustomer
    case dirty
    case cleaning
    case repairing
    case empty
    
    var description : String {
        switch self {
        case .occupiedByCustomer:
            return "occupiedByCustomer"
        case .dirty:
            return "dirty"
        case .cleaning:
            return "cleaning"
        case .repairing:
            return "repairing"
        case .empty:
            return "empty"
        }
      }
}
